README.md

```diff
@@ -126,20 +126,20 @@ Most of the unit tests in Logstash are written using [rspec](http://rspec.info/)
 
 ### Core tests
 
-1- In order to run the core tests, a small set of plugins must first be installed:
+1- To run the core tests you can use the rake task:
```

Comment:
did you mean gradle task ?

Reply:
@jakelandis hmm ... yes :D sec fixing :)